cfn-template/deeplearning.template

```diff
@@ -150,7 +168,7 @@
         "Role": { "Fn::GetAtt" : ["LambdaExecutionRole", "Arn"] },
         "Code": {
             "S3Bucket":  {"Fn::Join" : ["", [{ "Ref" : "AWS::Region" }, { "Fn::FindInMap" : [ "Other", "S3SourceBucket", "BucketNameSuffix" ]} ] ]},
-            "S3Key": { "Fn::FindInMap" : [ "Other", "LambdaFunction", "FileName" ]},
+            "S3Key": { "Fn::FindInMap" : [ "Other", "LambdaFunction", "FileName" ]}
```

Comment:
i see you removed the `,` at the end of this line ? is this intentional?

Reply:
@nswamy yes, last object according to the json spec should not end with a coma (even though AWS parsers, looks like, are capable of ignoring such error)